### PR TITLE
Add userid related challenge questions APIs

### PR DESCRIPTION
**Pull request by IsuraD**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/develop/restapis/challenge.yaml [hunk 1]
```diff
@@ -12,12 +12,17 @@ info:
...
+  - https
 
 # Tags are used for organizing operations
 tags:
-- name: me
-  description: Operations for the authenticated user.
+  - name: me
+    description: Operations for the authenticated user.
+  - name: admin
+    description: >
```
Comment (tharindu-bandara) [inline en/docs/develop/restapis/challenge.yaml]:
```suggestion
    description: |
```

File: en/docs/develop/restapis/challenge.yaml [hunk 2]
```diff
@@ -168,7 +173,7 @@ paths:
       summary: Answers a specific new challenge.
       operationId: addChallengeAnswerOfLoggedInUser
       description: |
-        Provide an **answer** to **a specific challenge** in the system for a logged in user. The user can at the most select one question from a challenge set of interest.
+        Provide an **answer** to **a specific challenge** in the system for loggedin user. The user can at most select one question from a challenge set of interest.
```
Comment (tharindu-bandara) [inline en/docs/develop/restapis/challenge.yaml]:
```suggestion
        Provide an **answer** to **a specific challenge** in the system for logged-in user. The user can at most select one question from a challenge set of interest.
```

File: en/docs/develop/restapis/challenge.yaml (hunk 1 above)
Comment (IsuraD) [reply en/docs/develop/restapis/challenge.yaml]: This should be fine

File: en/docs/develop/restapis/challenge.yaml [hunk 3]
```diff
@@ -12,12 +12,17 @@ info:
...
 tags:
-- name: me
-  description: Operations for the authenticated user.
+  - name: me
+    description: Operations for the authenticated user.
+  - name: admin
+    description: >
+      Operations available for privileged users, to be invoked on behalf of another user.
+
+      **These endpoints are not released in WSO2 Identity Server - 5.9.0 version and will be released in the next product version.**
```
Comment (madurangasiriwardena) [inline en/docs/develop/restapis/challenge.yaml]: Since we are not adding this to 5.9.0 doc space, I think this is not needed.

**Pull request merged**